docs/src/administration/web/environments.md

```diff
@@ -16,14 +16,12 @@ The relationships between environments is hierarchical, and can be organized in
 
 You can have branches that are not tied to a running instance of your application; these are what we call "inactive environments".
 
-## Master environment
+## Parent environment
 
-Every Platform.sh project starts with a Master environment which corresponds to the Master branch in Git.
-
-If you subscribed to a production plan, this environment is your **live site** and can be mapped to a domain name and a custom SSL certificate.
+Every Platform.sh project starts with a parent environment. It is Master by default, and corresponds to the `master` branch in Git. If you subscribed to a production plan, this environment is your **live site** and can be mapped to a domain name and a custom SSL certificate.
 
 {{< note >}}
-Your project must have a `master` branch: it will not function properly without one.
+Your project must have a parent environment, but you can configure it to be a different branch than master. See the ["Renaming a project's parent environment"](/guides/general/default-branch.md) guide for more details.
```

Comment:
```suggestion
Your project must have a parent environment, but you can configure it to be a different branch than master. See the ["Renaming a project's root environment"](/guides/general/default-branch.md) guide for more details.
```

Reply:
Or maybe use default for both?  "parent" doesn't work here, since parent is relative.

Reply:
In the console it's described as "Master is the main" environment, just didn't want that to get confusing with `main`. I'm good with root.

Reply:
Default? The actual parameter name is default_branch